src/Components/GeneralComponents/GeneralWindow.css

```diff
@@ -1,3 +1,43 @@
-.generalWindowContentContainer {
+.generalWindow {
+  background-color: #ddcc66;
+  text-align: right;
+  padding: 7px;
+  border-radius: 5px;
+  z-index: 1;
+}
+
+.centered {
+  width: fit-content;
+  position: fixed;
+  top: 52%; /* simply wanted it down a bit for astetics, thats whats behind the 2%*/
+  left: 50%;
+  transform: translate(-50%, -50%);
+  /*  the transform makes the window's anchor point to be in it's center instead of top left,
+        that way it is possible to center it easily instead of calculating its position according
+        to it's dimensions   */
+}
+
+.rightSided {
+  width: fit-content;
+  position: fixed;
+  right: 100px;
+  top: 52%;
+  transform: translate(0, -50%);
+  /*  same case as the comment above, instead this time I wanted it's anchor point to be on
```

Comment:
width and position could go to .generalWindow
also - is the position fixed necessary?